documentation/book/con-kafka-listeners.adoc

```diff
@@ -123,4 +123,48 @@ listeners:
 ----
 
 
-Authentication must be configured when using the User Operator to manage `KafkaUsers`.
\ No newline at end of file
+Authentication must be configured when using the User Operator to manage `KafkaUsers`.
+
+== Network Policies
+
+For every enabled listener, {ProductName} will automatically create a NetworkPolicy which will by default grant all applications access to the listener.
```

Comment:
_Please reword the opening paragraph as follows:_

AMQ Streams automatically creates a `NetworkPolicy` resource for every listener that is enabled on a Kafka broker. By default, a `NetworkPolicy` grants access to a listener to all applications and namespaces. If you want to restrict access to a listener to only selected applications or namespaces, use the `networkPolicyPeers` field. Each listener can have a different `networkPolicyPeers` configuration.